src/main/resources/application.yaml

```diff
@@ -11,33 +11,30 @@ management:
       exposure:
         include: health, info, prometheus
 
-#If you use a database then uncomment below lines and update db properties accordingly
 spring:
   config:
     import: "optional:configtree:/mnt/secrets/"
   application:
-    name: Spring Boot Template
-#  datasource:
-#    driver-class-name: org.postgresql.Driver
-#    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${DB_OPTIONS:}
-#    username: ${DB_USER_NAME}
-#    password: ${DB_PASSWORD}
-#    properties:
-#      charSet: UTF-8
-#    hikari:
-#      minimumIdle: 2
-#      maximumPoolSize: 10
-#      idleTimeout: 10000
-#      poolName: {to-be-defined}HikariCP
-#      maxLifetime: 7200000
-#      connectionTimeout: 30000
-#  jpa:
-#    properties:
-#      hibernate:
-#        jdbc:
-#          lob:
-#            # silence the 'wall-of-text' - unnecessary exception throw about blob types
-#            non_contextual_creation: true
+    name: Case Payment Orders API
+  datasource:
+    url: jdbc:postgresql://${CPO_CASE_PAYMENT_ORDERS_DB_HOST:localhost}:${CPO_CASE_PAYMENT_ORDERS_DB_PORT:6968}/${CPO_CASE_PAYMENT_ORDERS_DB_NAME:cpo_case_payment_orders}${CPO_CASE_PAYMENT_ORDERS_DB_OPTIONS:}
+    username: ${CPO_CASE_PAYMENT_ORDERS_DB_USERNAME:cpo}
+    password: ${CPO_CASE_PAYMENT_ORDERS_DB_PASSWORD:cpo}
+    type: com.zaxxer.hikari.HikariDataSource
+  jpa:
+    database: postgresql
+    show-sql: false
+    hibernate:
+      ddl-auto: none
+  flyway:
+    baseline-on-migrate: true
```

Comment:
baseline properties not needed as this is new DB.

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/3f932e0a934d18ed4fb8e61d94ed18b6058a32f9